test/e2e_installer/cluster/cluster_test.go

```diff
@@ -108,19 +108,19 @@ var _ = Describe("cluster", func() {
 		Expect(cls.Spec.Type).Should(Equal("Imported"), "Cluster type was not 'Imported'")
 	})
 
-	DescribeTable("Upgrade cluster",
-		func(oldVersion, newVersion string) {
-			cls = testTKE.ClusterTemplate()
-			cls.Spec.Version = oldVersion
-			cls, err = testTKE.CreateClusterInternal(cls)
-			Expect(err).To(BeNil(), "Create cluster failed")
-
-			cls, err = testTKE.UpgradeCluster(cls.Name, newVersion, platformv1.UpgradeModeAuto, false)
-			Expect(err).Should(BeNil(), "Upgrade cluster failed")
-			Expect(cls.Spec.Version).Should(Equal(newVersion), "Cluster version is wrong")
-		},
-		// Entry("1.19.7->1.20.4", "1.19.7", "1.20.4"),
-		Entry("1.20.6-tke.2->1.21.4-tke.3", "1.20.6-tke.2", "1.21.4-tke.3"))
+	// DescribeTable("Upgrade cluster",
+	// 	func(oldVersion, newVersion string) {
+	// 		cls = testTKE.ClusterTemplate()
+	// 		cls.Spec.Version = oldVersion
+	// 		cls, err = testTKE.CreateClusterInternal(cls)
+	// 		Expect(err).To(BeNil(), "Create cluster failed")
+
+	// 		cls, err = testTKE.UpgradeCluster(cls.Name, newVersion, platformv1.UpgradeModeAuto, false)
+	// 		Expect(err).Should(BeNil(), "Upgrade cluster failed")
+	// 		Expect(cls.Spec.Version).Should(Equal(newVersion), "Cluster version is wrong")
```

Comment:
I perfer to use "/* */" to comments block of code

Reply:
done